diff --git a/vendor/facebook/php-business-sdk/src/FacebookAds/Object/ServerSide/Content.php b/vendor/facebook/php-business-sdk/src/FacebookAds/Object/ServerSide/Content.php
index 8ca31c8..798d45e 100644
--- a/vendor/facebook/php-business-sdk/src/FacebookAds/Object/ServerSide/Content.php
+++ b/vendor/facebook/php-business-sdk/src/FacebookAds/Object/ServerSide/Content.php
@@ -205,7 +205,7 @@ class Content implements ArrayAccess {
    * @param integer $offset Offset
    * @return boolean
    */
-  public function offsetExists($offset) {
+  public function offsetExists($offset):bool {
     return isset($this->container[$offset]);
   }

@@ -214,7 +214,7 @@ class Content implements ArrayAccess {
    * @param integer $offset Offset
    * @return mixed
    */
-  public function offsetGet($offset) {
+  public function offsetGet($offset):bool {
     return isset($this->container[$offset]) ? $this->container[$offset] : null;
   }

@@ -224,7 +224,7 @@ class Content implements ArrayAccess {
    * @param mixed $value Value to be set
    * @return void
    */
-  public function offsetSet($offset, $value) {
+  public function offsetSet($offset, $value):void {
     if (is_null($offset)) {
       $this->container[] = $value;
     } else {
@@ -237,7 +237,7 @@ class Content implements ArrayAccess {
    * @param integer $offset Offset
    * @return void
    */
-  public function offsetUnset($offset) {
+  public function offsetUnset($offset):void {
     unset($this->container[$offset]);
   }

--
2.34.1

