diff --git a/vendor/facebook/php-business-sdk/src/FacebookAds/Object/ServerSide/Event.php b/vendor/facebook/php-business-sdk/src/FacebookAds/Object/ServerSide/Event.php
index ba99ada..ad66716 100644
--- a/vendor/facebook/php-business-sdk/src/FacebookAds/Object/ServerSide/Event.php
+++ b/vendor/facebook/php-business-sdk/src/FacebookAds/Object/ServerSide/Event.php
@@ -322,7 +322,7 @@ class Event implements ArrayAccess {
    * @param integer $offset Offset
    * @return boolean
    */
-  public function offsetExists($offset) {
+  public function offsetExists($offset):bool {
     return isset($this->container[$offset]);
   }

@@ -331,7 +331,7 @@ class Event implements ArrayAccess {
    * @param integer $offset Offset
    * @return mixed
    */
-  public function offsetGet($offset) {
+  public function offsetGet($offset):mixed {
     return isset($this->container[$offset]) ? $this->container[$offset] : null;
   }

@@ -341,7 +341,7 @@ class Event implements ArrayAccess {
    * @param mixed $value Value to be set
    * @return void
    */
-  public function offsetSet($offset, $value) {
+  public function offsetSet($offset, $value):void {
     if (is_null($offset)) {
       $this->container[] = $value;
     } else {
@@ -354,7 +354,7 @@ class Event implements ArrayAccess {
    * @param integer $offset Offset
    * @return void
    */
-  public function offsetUnset($offset) {
+  public function offsetUnset($offset):void {
     unset($this->container[$offset]);
   }

--
2.34.1

